index.html

```diff
@@ -5,6 +5,11 @@
 <img src="images/workshop-large.jpg" class="hero" alt="people working together on computers"></img>
 <div class="container">
     <h1 class="headline">We build bridges.</h1>
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+          Empower people with technology through teaching and facilitating access, enlarging the community of people who give back and teach others.
```

Comment:
I like: "We build bridges by empowering people in technology by teaching and facilitating access as well as by growing the community of volunteers and leaders."


On Feb 19, 2016, at 10:34 AM, Isa <notifications@github.com> wrote:

Empower people with technology through teaching and facilitating access, enlarging the community of people who give back and teach others.